app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/returns/ConvertedPackagingCreditController.scala

```diff
@@ -74,7 +84,9 @@ class ConvertedPackagingCreditController @Inject() (
         .bindFromRequest()
         .fold(
           (formWithErrors: Form[ConvertedPackagingCreditDetails]) =>
-            Future.successful(BadRequest(page(formWithErrors))),
+            exportCreditBalanceAvailable().map { balanceAvailable =>
+              BadRequest(page(formWithErrors, balanceAvailable))
+            },
```

Comment:
Can't get the comment attached exactly where I'd like it to be but should we be checking that the entered amount is not in excess of the balance available?

Reply:
In fact it might make more sense to have the credit available associated with the TaxReturn object like we've done for the oldest obligation?

Reply:
The scope of this ticket was just to trigger the API call and get the balance on the screen. So no validation and no incorporating the balance into the bigger picture - that's more for returns v2.

Reply:
👍